repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/keystonev2/selfvalidation/KeystoneV2SelfValidationTest.groovy

```diff
@@ -54,7 +57,10 @@ class KeystoneV2SelfValidationTest extends ReposeValveTest {
     }
 
     def setup() {
+        fakeIdentityV2Service.resetCounts()
         fakeIdentityV2Service.resetDefaultParameters()
+        fakeIdentityV2Service.resetHandlers()
+        sleep(500)
```

Comment:
We have like 3 duplicate stories for this:
https://repose.atlassian.net/browse/REP-4457